.github/workflows/ci.yml

```diff
@@ -131,23 +130,95 @@ jobs:
           curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
       - name: run golangci-lint
         run: golangci-lint run ./...
-
+  
   golang-unit-tests:
     runs-on: ${{ matrix.os }}
-    timeout-minutes: 15
     needs: golang-build
     strategy:
       matrix:
-        os: [ubuntu-20.04]
+        os: [ ubuntu-20.04 ]
     steps:
+        # Checkout and tool to format the test output
       - uses: actions/checkout@v3
       - uses: ./.github/actions/alicenet-config
-      # tool to format the test output
       - name: Set up gotestfmt
         run: go install github.com/haveyoudebuggedit/gotestfmt/v2/cmd/gotestfmt@latest
-      # Run tests with nice formatting. Save the original log in /tmp/gotest.log
-      # packages where the tests are stuck: ["blockchain", "badgerTrie", "consensus", "transport"]
-      - name: Run tests
+      - name: Run unit tests
+        timeout-minutes: 20
         run: |
           set -euo pipefail
           go test -json -v $(go list ./... | grep -Ev '/blockchain|/badgerTrie|/consensus|/testutils') 2>&1 | tee /tmp/gotest.log | gotestfmt
+
+  golang-blockchain-tests:
+    runs-on: ${{ matrix.os }}
+    needs: golang-build
+    timeout-minutes: 60
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-20.04 ]
+        test-cmd: [
+          github.com/MadBase/MadNet/blockchain/dkg/dkgtasks -run TestShareDistribution_Group_1,
```

Comment:
Are these not going to run the tests in the remote package?

Reply:
Running using the github package it's not going to download the package from github master and run from there? Or is it going to grab the local files?

Reply:
I guess it's running the local files as in this case `TestShareDistribution_Group_1` won't exist on `master` branch.